client/src/app/AppTypes.res

```diff
@@ -55,7 +55,7 @@ module CanvasProps = {
 module PageVisibility = {
   @ppx.deriving(show({with_path: false}))
   type rec t =
-    | Hidden
+    | Hidden(Js.Date.t)
```

Comment:
Does this seem reasonable - storing the "hidden since [x]" date in the `PageVisibility`? It works out well I think.